examples/DotNetCore/AzureFunction/FunctionAppIsolatedMode/AzureAppConfigurationRefreshExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Hosting;
+
+namespace FunctionAppIsolatedMode
+{
+    internal static class AzureAppConfigurationRefreshExtensions
+    {
+        public static IFunctionsWorkerApplicationBuilder UseAzureAppConfiguration(this IFunctionsWorkerApplicationBuilder builder)
+        {
+            if (builder == null)
```

Comment:
I don't expect argument validation on the extended object an extension method considering it's usage.

Reply:
Removed